testfiles/1ZoneUncontrolled3SurfaceZone.idf

```diff
@@ -0,0 +1,397 @@
+!-Generator IDFEditor 1.34
+!-Option OriginalOrderTop UseSpecialFormat
+!-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
+!-      Use '!' comments if they need to be retained when using the IDFEditor.
+!1ZoneUncontrolled3SurfaceZone.idf
```

Comment:
@mjwitte My intent was not to add a file to the test suite just for this error message so I think its inclusion in the commit was an error on my part.  The point was basically to have a file that I ran that showed the error message.  However, we don't need to clutter up the test suite for that since we have a unit test that does that now.  Did it get added when you merged?